packages/paste-core/components/toast/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@twilio-paste/toast",
+  "version": "0.0.0",
+  "category": "feedback",
+  "status": "beta",
+  "description": "A toast communicates an immediate and direct response to a user action.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && yarn compile",
+    "build:dev": "yarn clean && yarn compile:dev",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.build.tsbuildinfo && rm -rf .rpt2_cache",
+    "compile": "rollup -c --environment NODE_ENV:production",
+    "compile:dev": "rollup -c --environment NODE_ENV:development",
+    "prepublishOnly": "yarn build",
+    "type-check": "tsc --noEmit"
+  },
+  "peerDependencies": {
+    "@twilio-paste/absolute": "^2.0.70",
```

Comment:
I don't think you're using absolute, is it a peer dependency of another thing you're using?

Reply:
Right. I had added because I thought there was a fixed variant, then forgot to remove it.